indico/htdocs/sass/modules/_registrationform.scss

```diff
@@ -1304,3 +1304,70 @@ textarea.regFormDescriptionInputMgmt {
         }
     }
 }
+
+
+.invitation-list {
+    .invitation-table {
```

Comment:
Couldn't you reuse the style we had for agreements? I think it's pretty similar.